src/app/ApplicationTemplate.Shared.Views/Framework/Email/EmailImplementationExtensions.Windows.cs

```diff
@@ -0,0 +1,30 @@
+﻿#if __WINDOWS__
+#nullable enable
+using System.Threading.Tasks;
+
+namespace ApplicationTemplate;
+
+/// <summary>
+/// Static class with extension methods for the <see cref="IEmail"/> APIs.
+/// </summary>
+public static class EmailImplementationExtensions
```

Comment:
The extensions are on `IEmail` and I'm not sure to know why they would only be available on Windows?

Reply:
Ohh must be left over, I tought I rebased, weird.